### PR TITLE
Add exam stats function

### LINKED ISSUE #144: As an expert user, I can view my students' academic progress
... so that I know which students need more help.

### PR DESCRIPTION
**Pull request by hogantan**
Brief Description of Exam Stats function:
* Part of Exam Command feature
* View exam statistics of a selected student (graphical representation of exam scores)
* Opens a new window upon command call

Screenshot:
image

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -10,29 +10,30 @@
...
+        + [3.3.1.2 Resolving a question: `question solve`](#3312-resolving-a-question---question-solve-)
+        + [3.3.1.3 Deleting a question: `question delete`](#3313-deleting-a-question---question-delete-)
+    + [3.3.2 Recording exams of a student: `exam` (By: Hogan)](#332-recording-exams-of-a-student---exam---by--hogan-)
+        + [3.3.2.1 Adding an exam record to a student: `exam add`](#3321-adding-an-exam-record-to-a-student---exam-add-)
+        + [3.3.2.2 Deleting an exam record for a student: `exam delete`](#3322-deleting-an-exam-record-for-a-student---exam-delete-)
+        + [3.3.2.3 Viewing exam statistics of a student: `exam stats`](#3323-viewing-exam-statistics-of-a-student---exam-stats-)
+    + [3.3.3 Recording attendance of a student: `attendance` (By: Vaishak)](#333-recording-attendance-of-a-student---attendance---by--vaishak-)
+        + [3.3.3.1 Adding an attendance record to a student: `attendance add`](#3331-adding-an-attendance-record-to-a-student---attendance-add-)
+        + [3.3.3.2 Deleting an attendance record for a student: `attendance delete`](#3332-deleting-an-attendance-record-for-a-student---attendance-delete-)
```
Comment (VaishakAnand) [inline docs/UserGuide.md]: Why are these changed uh, cos i think the extra `---` in the link breaks the links.

**Review by VaishakAnand — CHANGES_REQUESTED** [review]
The command part seems fine, I think just need to reuse the same links from the current user guide, then just add the extra exam stats section and link.

**Review by VaishakAnand — APPROVED** [review]

File: docs/UserGuide.md (hunk 1 above)
Comment (hogantan) [reply docs/UserGuide.md]: Oh yea my bad. Not sure what led me to add those extra `---`. Thanks for pointing it out! I have reverted accordingly.
